extensions/amp-base-carousel/0.1/amp-base-carousel.js

```diff
@@ -240,9 +261,9 @@ class AmpCarousel extends AMP.BaseElement {
     const html = htmlFor(this.element);
     return html`
       <div class="i-amphtml-carousel-content">
-        <div class="i-amphtml-carousel-scroll"></div>
-        <div class="i-amphtml-carousel-arrow-next-slot"></div>
         <div class="i-amphtml-carousel-arrow-prev-slot"></div>
+        <div class="i-amphtml-carousel-arrow-next-slot"></div>
+        <div class="i-amphtml-carousel-scroll"></div>
```

Comment:
One of the carousel accessibility guides I was reading said that the buttons should come before the content. I'm not entirely clear on why that is and I can't find it anymore. Putting this back to the way it was (but still keeping the CSS changes in case we want to move them again).